server/src/gql/resolver.py

```diff
@@ -0,0 +1,60 @@
+"""GraphQL Ariadne resolver functions."""
+# 'annotations' import needs to return enclosing type:
+# https://www.python.org/dev/peps/pep-0563/
+from __future__ import annotations
```

Comment:
doesn't look like you're returning enclosing type here either.

Reply:
ditto